docs/clients/ssm.md

```diff
@@ -23,7 +23,6 @@ $parameters = $ssm->getParametersByPath(new GetParametersByPathRequest([
 ]));
 
 $secrets = [];
-/** @var Parameter $parameter */
```

Comment:
We could also remove use statement for Parameter

Reply:
good catch!